components/article-list.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import ArticleListItem from "./article-list-item";
+
+const List = (props) =>{
```

Comment:
You can use destructuring assignment `({ articles })`instead of `(props)` here to access articles right away.